modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -0,0 +1,147 @@
+/singleton/reagent/inaprovaline
+	description = "Inaprovaline is a chemical that narrows blood vessels, raises blood pressure, and helps the blood pump more efficiently. Commonly used to stabilize patients."
+
+/singleton/reagent/bicaridine
+	description = "Bicaridine is an analgesic medication and can be used to treat blunt trauma. Lasts twice as long when inhaled, however it is generally twice as weak."
+
+/singleton/reagent/kelotane
+	description = "Kelotane is a drug used to treat burns."
+
+/singleton/reagent/dexalin
+	description = "Dexalin is used in the treatment of oxygen deprivation. The medication is twice as powerful and lasts twice as long when inhaled."
+
+/singleton/reagent/dexalin/plus
+	description = "Dexalin Plus is used in the treatment of oxygen deprivation. It is highly effective, and is twice as powerful and lasts twice as long when inhaled."
+
+/singleton/reagent/tricordrazine
+	description = "Tricordrazine is a highly potent stimulant, originally derived from cordrazine. Can be used to treat a wide range of injuries, however it does not work when inhaled. Has different healing properties depending on the chemical's temperature."
```

Comment:
"Tricordrazine is an old, though still useful, medication that can be used to treat a wide range of injuries, though at a slower rate than any specialized chemical. The drug increases the rate at which tissues regenerate and has different regenerative properties depending on the patient´s body temperature. Does not work when inhaled."